packages/react-native-sdk/src/components/call/CallControls/AcceptCallButton.tsx

```diff
@@ -0,0 +1,58 @@
+import { useCall } from '@stream-io/video-react-bindings';
+import React, { useCallback } from 'react';
+import { CallControlsButton } from './CallControlsButton';
+import { theme } from '../../../theme';
+import { Phone } from '../../../icons';
+
+/**
+ * The props for the Accept Call button.
+ */
+type AcceptCallButtonProps = {
+  /**
+   * Handler to be called when the accept call button is pressed.
+   * @returns void
+   */
+  onPressHandler?: () => void;
+  /**
+   *  Handler to be called after the incoming call is accepted.
+   */
+  onAcceptHandler?: () => void;
+};
+
+/**
+ * Button to accept a call.
+ *
+ * Mostly calls call.join() internally.
+ */
+export const AcceptCallButton = ({
+  onPressHandler,
+  onAcceptHandler,
+}: AcceptCallButtonProps) => {
+  const call = useCall();
+  const acceptCallHandler = useCallback(async () => {
```

Comment:
we can remove this useCallback wrapper, unnecessary here

Reply:
Done ✅